src/components/InventoryDetail/ApplicationDetails.js

```diff
@@ -14,11 +14,14 @@ const ApplicationDetails = ({ onTabSelect, appList, ...props }) => {
     const history = useHistory();
     const dispatch = useDispatch();
     const searchParams = new URLSearchParams(search);
-    const items = useSelector(({ entityDetails }) => entityDetails?.activeApps);
+    const items = useSelector(({ entityDetails }) => entityDetails?.activeApps.filter(({ isVisible }) => isVisible !== false));
     const activeApp = useSelector(({ entityDetails }) => entityDetails?.activeApp);
+    const disabledApps = useSelector(({ systemProfileStore }) => systemProfileStore?.disabledApps);
     const defaultApp = activeApp?.appName || appList?.find(({ pageId, name }) => items?.[0]?.name === (
         pageId || name))?.name || items?.[0]?.name;
-    const applications = appList || items;
+    let applications = appList || items;
+    const [activeTabs, setActiveTabs] = useState(applications);
```

Comment:
This can be simplified to directly assign to default state
```suggestion
    const [activeTabs, setActiveTabs] = useState(appList || items);
```